Description: Remove reference to opj_free for now
Author: Mathieu Malaterre <malat@debian.org>
Cmment: Thanks to Valerii Zapodovnikov for update
Last-Update: 2021-01-27

Index: openjpeg2/src/bin/jp3d/convert.c
===================================================================
--- openjpeg2.orig/src/bin/jp3d/convert.c
+++ openjpeg2/src/bin/jp3d/convert.c
@@ -581,7 +581,7 @@ opj_volume_t* bintovolume(char *filename
     f = fopen(filename, "rb");
     if (!f) {
         fprintf(stdout, "[ERROR] Failed to open %s for reading !!\n", filename);
-        opj_free(volume);
+        free(volume);
         return 0;
     }
 
@@ -919,7 +919,7 @@ opj_volume_t* imgtovolume(char *fileimg,
     f = fopen(filename, "rb");
     if (!f) {
         fprintf(stderr, "[ERROR] Failed to open %s for reading !!\n", filename);
-        opj_free(volume);
+        free(volume);
         return 0;
     }
 
